vault/dendron.tutorial.taking-notes.md

```diff
@@ -2,53 +2,68 @@
-updated: 1636527192313
+updated: 1639196680642
 created: 1625563944736
 nav_order: 1
 ---
 
 ### Create a Note
 
-To create a note, use `Ctrl+L` / `Cmd+L` to bring up Dendron's lookup. This is a shortcut for the `Dendron: Lookup` command.
+To create a note, use `Ctrl+L` / `Cmd+L` to bring up [[Dendron's Lookup|dendron://dendron.dendron-site/dendron.topic.lookup]]. This is a shortcut for the `Dendron: Lookup` command.
 
 > 💡 The lookup command is the main way to interact with Dendron. It is used for both looking up your notes and creating new notes. When you do a lookup on a note that hasn't been created, Dendron will create it for you. Remember the `Ctrl+L` / `Cmd+L` shortcut.
 
-> 🌱 Try it yourself - bring up the lookup bar with `Ctrl+L` / `Cmd+L`. Type `recipes` and hit Enter. Afterward, you can use the `Ctrl+Tab` / `Cmd+Tab` shortcut to get back to the previous note.
+> 🌱 Try it yourself! Open the lookup bar with `Ctrl+L` / `Cmd+L`. Type `recipes` and hit `Enter`. Afterward, you can use the `Ctrl+Tab` / `Cmd+Tab` shortcut to get back to the previous note you were viewing.
 
-This should have created a note named `recipes.md`. Notes in Dendron are just plaintext markdown. They live in your file system and are portable across any platform, which makes it very easy to import/ export your notes and even to publish your notes to a hosted website. You can try to add some content to the body of your `recipes` note now.
+This should have created a note named `recipes.md`. Notes in Dendron are just plaintext [[markdown|dendron://dendron.dendron-site/dendron.topic.markdown]]. They live in your file system and are portable across any platform, which makes it very easy to import/export your notes and even to [[publish your notes to a hosted website|dendron://dendron.dendron-site/dendron.topic.publish]]. You can try to add some content to the body of your `recipes` note now.
 
-The --- section at the top of each note is frontmatter. Frontmatter are custom attributes at the top of each markdown file. Dendron uses it to store metadata about each note for features like publishing. Don't modify the id attribute on the front matter.
+The `---` section at the top of each note is **frontmatter**. [[Frontmatter|dendron://dendron.dendron-site/dendron.topic.frontmatter]] contains custom attributes at the top of each markdown file. Dendron uses it to store metadata about each note for features like publishing.
+
+- Don't modify the `id` or `created` attributes in frontmatter.
+- The `updated` attribute automatically _updates_ whenever you make a new modificaiton to a file.
 
 ### Creating a hierarchy
 
-Dendron uses flexible hierarchies to help you organize your notes. It's how people are able to manage tens of thousands of notes inside Dendron.
+Dendron uses flexible hierarchies to help you organize your notes. [It's how people are able to manage tens of thousands of notes inside Dendron](https://www.kevinslin.com/notes/3dd58f62-fee5-4f93-b9f1-b0f0f59a9b64/).
```

Comment:
we should provide more context for any dendron specific jargon. can we add a definition for this in [[Concepts|concepts]] and link to it?

You can paraphrase my answer from the ness article: https://nesslabs.com/dendron-featured-tool

> Dendron has flexible hierarchies. They provide a structure for your notes but these structures can be easily changed. In programming, developers can refactor code and change its structure — the IDE will make sure that all references pointing to the original code are updated. In Dendron, you can refactor notes and hierarchies and Dendron will make sure that your PKM is consistent throughout. This means that you have the best of both worlds: a basic structure for the organization but the flexibility to change it.

Reply:
I've added to [[dendron.topic.hierarchies]], and did a note ref in [[dendron.concepts]] and here.